pkgs/development/python-modules/skops/default.nix

```
@@ -44,13 +45,18 @@ buildPythonPackage rec {
     streamlit
   ];
   pytestFlagsArray = [ "skops" ];
-  disabledTestPaths = [
-    # try to download data from Huggingface Hub:
-    "skops/hub_utils/tests"
-    "skops/card/tests"
-    # minor output formatting issue
-    "skops/card/_model_card.py"
-  ];
+  disabledTestPaths =
+    [
+      # try to download data from Huggingface Hub:
+      "skops/hub_utils/tests"
+      "skops/card/tests"
+      # minor output formatting issue
+      "skops/card/_model_card.py"
+    ]
+    ++ lib.optional stdenvNoCC.isDarwin [
+      # Segfaults on darwin
+      "skops/io/tests/test_persist.py"
+    ];
```

- use optionals
- use stdenv

```suggestion
    ++ lib.optionals stdenv.isDarwin [
      # Segfaults on darwin
      "skops/io/tests/test_persist.py"
    ];
```

---

Fixed.